source/adapters/opencl/command_buffer.cpp

```diff
@@ -26,8 +26,8 @@ UR_APIEXPORT ur_result_t UR_APICALL urCommandBufferCreateExp(
           CLContext, cl_ext::ExtFuncPtrCache->clCreateCommandBufferKHRCache,
           cl_ext::CreateCommandBufferName, &clCreateCommandBufferKHR);
 
-  if (!clCreateCommandBufferKHR || Res != CL_SUCCESS)
-    return UR_RESULT_ERROR_INVALID_OPERATION;
+  if (Res != CL_SUCCESS)
+    return Res;
```

Comment:
this is returning a CL error code where there should be a UR one, instead we should remove this altogether and wrap the `getExtFunc` calls in `UR_RETURN_ON_FAILURE` (applies elsewhere)

Reply:
Good eye! This should have been addressed.